docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  notebook:
+    build:
+      context: .
+      args:
+        USER_UID: ${USER_UID:-1000}
+        USER_GID: ${USER_GID:-1000}
+    ports:
+      - "127:0.0.1:8888:8888"
```

Comment:
> Theres's a little fix in "docker-compose.yml" to make it launch with docker compose:
> erase the ip address in the port section like below
> ports:
>    "8888:8888"

@8huit normally this is intended, I don't want to expose jupyter to the local network but only to my machine.

Is this not working on macOS (I'm a bit surprise because I use this settings on a lot of other projects without problem.

Reply:
or maybe you have this issue: https://github.com/docker/for-mac/issues/5809#issuecomment-881148183 ?

Reply:
I have this issue on docker destop on my mac. Maybe it's specific, but i had to strip the ip address to make it run.
Otherwise, it works fine!

Reply:
I will merge for now, let see if other have the issue.